server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/http/Request.java

```diff
@@ -0,0 +1,18 @@
+package com.amazonaws.ml.mms.servingsdk.http;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Request {
+    List<String> getHeaderNames();
+    String getRequestURL();
+    String getRequestURI();
+    String getRequestPath();
+    Map<String, List<String>> getParameterMap();
+    List<String> getParameter(String k);
+    String getContentType();
+    String getRequestedSessionId();
+    String method();
```

Comment:
String getMethod()
to be consistence with other method

Reply:
getMethod was deprecated in the netty. I think thats because it might seem like its "GET" HTTP method.

Reply:
in netty, they move to new style of getter/setter, the getter doesn't have get any more.
But in our API, we use traditional getter/setter way

Reply:
True. We can go either ways. I removed it for now as we don't need it. I will plan on adding it back once we support generic HTTP methods.